hi-ml-histopathology/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Using the histopathology models
+
+## Onboarding to Azure
+
+Prio 1, to be completed later
+
+## Creating datasets
+
+* [How to get PANDA (whole slides) and upload it to Azure](panda_dataset.md)
+* [How to get TCGA-Crck (tiles) and upload it to Azure](tcga-crck_dataset.md)
+
+## Training a model
+
+* Prio 0: [PANDA DeepMil on whole slides with ImageNet encoder](panda_model.md)
+  * How to invoke training for the existing model on PANDA data
```

Comment:
what do we mean here with model? pre-trained encoder? I don't think we said to release a pre-trained deepmil model but I might have missed some steps here. Or maybe model here = pipeline?

Reply:
I think @ant0nsc means to train our existing PANDA _config_ from scratch.

Reply:
rephrased.